iroha/src/wsv.rs

```diff
@@ -1,7 +1,7 @@
 use crate::prelude::*;
 use std::collections::HashMap;
 
-#[derive(Debug, Default)]
+#[derive(Debug, Clone, Default)]
```

Comment:
I haven't found where you `clone` WSV we usually clone `Arc` that references WSV, so do we really need it to be `Clone`?

Reply:
We clone the whole WSV in `validate` as WA for now